indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -3244,6 +3275,40 @@ private void checkCurrentTaskState() throws ExecutionException, InterruptedExcep
     Futures.successfulAsList(futures).get(futureTimeoutInSeconds, TimeUnit.SECONDS);
   }
 
+  private void checkIfStreamInactiveAndTurnSupervisorIdle()
+  {
+    if (!spec.getSupervisorStateManagerConfig().isEnableIdleBehaviour()
+        || !spec.getIoConfig().isEnableIdleBehaviour()) {
+      return;
+    }
+
+    Map<PartitionIdType, SequenceOffsetType> latestSequencesFromStream = getLatestSequences();
+    long nowTime = Instant.now().toEpochMilli();
+    boolean idle;
+    if (lagsVerifiedlastTime > 0
+        && previousPartitionOffsetsSnapshot.equals(latestSequencesFromStream)
+        && computeTotalLag() == 0) {
+      idleTime += nowTime - lagsVerifiedlastTime;
```

Comment:
Isn't this simply an assignment? Why is the difference being added each time?

Reply:
It is used to compare with `ioConfig.idleSupervisorForStreamIdleMillis`, only if greater to turn Supervisor idle.

Reply:
Oh I'm sorry assignment for `lagsVerifiedlastTime` was done inside by mistake.